packages/@okta/vuepress-theme-prose/assets/css/components/_searchBar.scss

```diff
@@ -66,6 +76,12 @@ $redesign_placeholder_color: #616161;
             color: $redesign_placeholder_color;
           }
 
+          @media (max-width: 1080px) {
+            &::placeholder {
+              color: transparent;
```

Comment:
Is there any way we can combine this with the next rule applying to the same pseudo-element? They seem overlapping.

Reply:
Actually the 1080px would cover the <desktop (1024px) so I could just replace it.